KomaMRIBase/src/timing/KeyValuesCalculation.jl

```diff
@@ -32,6 +32,14 @@ get_theo_A(g::Grad; off_val=0) = begin
 	else
 		aux = [off_val; 0; A; 0]
 	end
+    # To remove abrut changes in the plots
+    if g.rise == 0
+        aux[2] = aux[3]
+    end
+    if g.fall == 0
+        aux[end] = aux[end-1]
+    end
+    # If no signal, then don't show samples
```

Comment:
If we change the `Grad(A,T,dt_gt/2,dt_gr/2,delay)` to `Grad(A,T,0,0,delay+dt_gr/2)` this would always be the case. Meaning that it would not be a special case. We need to add tests to verify that all the points are exactly like in pulseq, related to https://github.com/JuliaHealth/KomaMRI.jl/issues/224#issuecomment-1830076604 and #318, but for this, I would create some very simple sequences with [all of this](https://juliahealth.org/KomaMRI.jl/stable/events/) cases covered.